src/hotspot/share/oops/instanceKlass.hpp

```diff
@@ -255,8 +254,7 @@ class InstanceKlass: public Klass {
     _misc_has_nonstatic_concrete_methods      = 1 << 5,  // class/superclass/implemented interfaces has non-static, concrete methods
     _misc_declares_nonstatic_concrete_methods = 1 << 6,  // directly declares non-static, concrete methods
     _misc_has_been_redefined                  = 1 << 7,  // class has been redefined
-    _misc_has_passed_fingerprint_check        = 1 << 8,  // when this class was loaded, the fingerprint computed from its
-                                                         // code source was found to be matching the value recorded by AOT.
+    _unused                                   = 1 << 8,  //
```

Comment:
Any particular reason we don't want to remove this gap?

Reply:
Less changes. We don't get any benefits from removing it. It is opposite - if we need a new value we will use it without changing following values again.